src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -36,12 +44,16 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 	const url = new URL(`${req.protocol}://${req.get("host")}${req.originalUrl}`);
 	// Only has cloud instance
 	if (servers.hasCloudServer && servers.gheServerInfos.length == 0) {
-		res.redirect(`/github/create-branch${url.search}`);
+		logger.info("redirecting to cloud.");
+		res.set("Authorization", req.headers.authorization);
```

Comment:
Sorry what is `res.set("Authorization"` for? Will the client js pick up the Auth value from the header?

Reply:
ahh good catch that was was first attempt to force it there. 1 moment